static/app/views/alerts/incidentRules/triggers/actionsPanel/index.tsx

```diff
@@ -483,4 +509,14 @@ const PerformActionsListItem = styled(StyledListItem)`
   line-height: 1.3;
 `;
 
+const MarginlessAlert = styled(Alert)`
```

Comment:
Maybe let's not style it and let it just look like a normal alert?

Was there a design we were going off of? Doesn't look quite right right now

Reply:
🤷‍♀️ I did the exact same thing we do for issue alerts.

Reply:
Oh wait really? The thing that looked a bit off right now to me were these corners

<img width="493" alt="image" src="https://user-images.githubusercontent.com/1421724/168659260-57aa650b-c649-49b4-8035-5b2cf2b26c74.png">

They probably should blend together

Reply:
If I leave the alert unstyled it looks like this:

<img width="538" alt="Screen Shot 2022-05-16 at 11 48 53 AM" src="https://user-images.githubusercontent.com/29959063/168661682-66da15d4-7efe-4f40-9010-47b066573ab5.png">

Reply:
Yeah, in this case I would probably just toss some margin on top of it, though it may look a bit detached from this now 😬 Let's see what @robinrendle thinks

Reply:
Okay even though I 100% copy/pasted it from issue alerts, this is what it looks like for them: 

<img width="413" alt="Screen Shot 2022-05-16 at 11 51 37 AM" src="https://user-images.githubusercontent.com/29959063/168662101-e372d3c0-ad58-483b-8acd-d7e57fed593a.png">

So I can try to make it look the same.

Reply:
Ahh yeah I see, yeah okay, let's def see if we can make it same!